src/integrationtests/java/com/aws/greengrass/integrationtests/ipc/AuthorizeClientDeviceActionTest.java

```diff
@@ -73,7 +73,7 @@ private static void authzClientDeviceAction(GreengrassCoreIPCClient ipcClient,
             throws Exception {
         AuthorizeClientDeviceActionResponseHandler handler =
                 ipcClient.authorizeClientDeviceAction(request, Optional.empty());
-        AuthorizeClientDeviceActionResponse response = handler.getResponse().get(10, TimeUnit.SECONDS);
+        AuthorizeClientDeviceActionResponse response = handler.getResponse().get(5, TimeUnit.SECONDS);
```

Comment:
I'm surprised this would take longer than a couple milliseconds. What's so slow?

Reply:
I don't see it being slow for any reason but how does that impact the tests? Are we trying to fail-fast if there's actually an issue that makes it take more than a few milliseconds?

Reply:
Failing fast, yes. But also, if it really is taking 1+ seconds, then that's a real issue.